module/apmhttp/ignorer.go

```diff
@@ -53,6 +54,16 @@ func DefaultServerRequestIgnorer() RequestIgnorerFunc {
 	return defaultServerRequestIgnorer
 }
 
+// DynamicServerRequestIgnorer returns the RequestIgnorer to use in
+// handlers. The list of wildcard patterns comes from central config
+func DynamicServerRequestIgnorer(t *apm.Tracer) RequestIgnorerFunc {
+	matchers := t.GetIgnoreURLs()
+	if len(matchers) != 0 {
+		defaultServerRequestIgnorer = NewWildcardPatternsRequestIgnorer(matchers)
+	}
+	return defaultServerRequestIgnorer
```

Comment:
```suggestion
	return func(r *http.Request) bool {
		matchers := t.GetIgnoreURLs()
		return matchers.MatchAny(r.URL.String())
	}
```

For it to be dynamic we'll need to fetch the matchers on each call, not once at construction time.